development/tsdb-blocks-storage-s3/docker-compose.yml

```diff
@@ -26,6 +26,14 @@ services:
     volumes:
       - .data-configstore:/usr/share/nginx/html/private:ro
 
+  prometheus:
+    image: prom/prometheus:v2.16.0
+    command: ["--config.file=/etc/prometheus/prometheus.yaml"]
+    volumes:
+      - ./config:/etc/prometheus
+    ports:
+      - 8010:8010
```

Comment:
Prometheus runs on 9090 right?

Reply:
Damn, yes! Thanks for spotting it! I was left there from the previous Grafana agent setup.